sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -0,0 +1,115 @@
+import random
+import unittest
+
+import pandas as pd
+
+from tests.integration.feature_repos.test_repo_configuration import (
+    Environment,
+    parametrize_online_test,
+)
+
+
+@parametrize_online_test
```

Comment:
is the intention to add more tests here? the original test seems to also e.g. test the TTL of the registry and to_df

Reply:
I'll add the to_df tests and registry tests as well

Reply:
Going to add this in a different PR.